server/py/framework/db/sqldb/models.py

```diff
@@ -935,16 +935,17 @@ class ModelEndpoint(Base, mlrun.utils.db.HasStruct):
         )
         function_id = Column(
             Integer,
-            ForeignKey("functions.id"),
+            ForeignKey("functions.id", ondelete="SET NULL"),
```

Comment:
Let's discuss if indeed "SET NULL" is the way to go..
Case in point: Deleting a model used by MEP might break the mm-apps if they use the model data for their work..